var/spack/repos/builtin/packages/openjpeg/package.py

```diff
@@ -56,3 +61,18 @@ def url_for_version(self, version):
     def libs(self):
         return find_libraries('libopenjp{0}'.format(self.version.up_to(1)),
                               root=self.prefix, recursive=True)
+
+    def cmake_args(self):
+        args = [
+            self.define_from_variant('BUILD_CODEC', 'codec'),
+            # MJ2 executables are disabled by default and we just make it
+            # explicit. Note that the executables require additional libraries
+            # as in the case '+codec', therefore, we will need to update the
+            # 'depends_on' directives when/if we introduce a variant that
+            # enables them.
+            self.define('BUILD_MJ2', False),
+            # Note that if the list of dependencies is incomplete, there is
+            # still a chance that the bundled third-party libraries get built.
+            self.define_from_variant('BUILD_THIRDPARTY', False)
```

Comment:
```suggestion
            self.define('BUILD_THIRDPARTY', False)
```
All hail CI for catching this bug!